erpnext/patches/v13_0/update_sticker_details_in_batch.py

```diff
@@ -0,0 +1,10 @@
+
+import frappe
+from erpnext.stock.doctype.batch.batch import update_batch_doc
+
+def execute():
+	frappe.reload_doc("stock", "doctype", "quality_inspection")
+	quality_inspection = frappe.get_all("Quality Inspection", fields=["name", "item_code", "batch_no", "thc", "cbd"])
```

Comment:
We only want submitted QIs:

```suggestion
	quality_inspection = frappe.get_all("Quality Inspection", filters={"docstatus": 1}, fields=["name", "item_code", "batch_no", "thc", "cbd"])
```